public/app/plugins/panel/geomap/editor/MapCenterEditor.tsx

```diff
@@ -23,53 +22,77 @@ export const MapCenterEditor: FC<StandardEditorProps<MapCenterConfig, any, Geoma
     return centerPointRegistry.selectOptions(ids);
   }, [value?.id]);
 
+  useEffect(() => {
+    if (!context.builder || !value) {
+      return;
+    }
+    if (value.id === MapCenterID.Coordinates) {
+      context.builder.addNumberInput({
+        path: 'view.center.lat',
+        category: ['Map View'],
+        name: 'Latitude',
+        showIf: (o) => o.view.center.id === MapCenterID.Coordinates,
+      });
+    }
+  });
```

Comment:
This is performed only on init - I think can be extracted to a reusable callback for use in Select's onChange